integration-tests/messages/addIPFSMessage.test.ts

```diff
@@ -29,45 +22,48 @@ describe("Add Offchain Message", function () {
     const ipfs_payload_len = ipfs_payload_data.length;
 
     before(async function () {
-        let connectApi = await connect(process.env.WS_PROVIDER_URL);
-        api = connectApi
-        fundingInputs = generateFundingInputs(api, this.title);
-
-        keys = (await createAndFundAccount(fundingInputs)).newAccount;
+        keys = await createAndFundKeypair();
 
         // Create a new MSA
-        await createMsa(api, keys);
+        const createMsa = ExtrinsicHelper.createMsa(keys);
+        await createMsa.fundAndSend();
 
         // Create a schema
-        const createSchemaEvents = await createSchema(api, keys, PARQUET_BROADCAST, "Parquet", "IPFS");
+        const createSchema = ExtrinsicHelper.createSchema(devAccounts[0].keys, PARQUET_BROADCAST, "Parquet", "IPFS");
```

Comment:
Maybe we should have a method for accessing the dev account instead of using an arbitrary int.

Reply:
We actually don't need dev account keys here, changed to use the keys we just created.